pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -729,23 +718,13 @@ def slitmask(self, tslits_dict, pad=None):
     @property
     def dloglam(self):
         # This number was computed by taking the mean of the dloglam for all the X-shooter orders. The specific
-        # loglam across the orders deviates from this value by +-7% from this first to final order
-        return 1.69207e-5
+        # loglam across the orders deviates from this value by +-7% from this first to final order. This is the
+        # unbinned value. It was actually measured  to be 1.69207e-5  from a 2x1 data and then divided by two.
+        return 8.46035e-06
 
     @property
     def loglam_minmax(self):
-        return np.log10(5000.0), np.log10(10500)
-
-    def wavegrid(self, binning=None, midpoint=False):
-
-        # Define the grid for VLT-XSHOOTER NIR
-        logmin, logmax = self.loglam_minmax
-        loglam_grid = utils.wavegrid(logmin, logmax, self.dloglam)
-        if midpoint:
-            loglam_grid = loglam_grid + self.dloglam/2.0
-
-        return np.power(10.0,loglam_grid)
-
+        return np.log10(5000.0), np.log10(11000)
```

Comment:
Same comment here about allowing this to be a parameter.

Reply:
Seem my point above. These are constants for the spectrograph that only developers should touch.